pkg/gameservers/gameservercounter.go

```diff
@@ -166,7 +169,7 @@ func (ac *AllocationCounter) Run(stop <-chan struct{}) error {
 }
 
 // Counts returns the NodeCount map in a thread safe way
-func (ac *AllocationCounter) Counts() map[string]NodeCount {
+func (ac *GameServerCounter) Counts() map[string]NodeCount {
```

Comment:
Nit: same as in the previous PR
```suggestion
func (gc *GameServerCounter) Counts() map[string]NodeCount {
```